rednose.py

```diff
@@ -366,7 +367,9 @@ def __init__(self, stream, excludes):
 		self.__stream = stream
 		self.__excludes = list(map(re.compile, excludes))
 
-	def __should_filter(self):
+	def __should_filter(self, a):
+		if a not in [('.',), ('X',), ('F',), ('S',)]:
+			return False
```

Comment:
I realized that after, but don't you only ever want to filter out the abbreviation and longer versions of the test status output? like `success` etc. What unknown output do you ever want to suppress?